isaric/parsers/recover-eu_parser.toml

```diff
@@ -0,0 +1,325 @@
+#:schema ../../schemas/dev/parser.schema.json
+
+[adtl]
+  name = "recover eu"
+  description = "recover EU consortium"
+
+  [adtl.tables.subject]
+    kind = "groupBy"
+    groupBy = "subject_id"
+    aggregation = "lastNotNull"
+    schema = "../../schemas/dev/subject.schema.json"
+
+  [adtl.tables.visit]
+    kind = "groupBy"
+    groupBy = "visit_id"
+    aggregation = "lastNotNull"
+    schema = "../../schemas/dev/visit.schema.json"
+    optional-fields = [ "outcome", "country_iso3" ]
+
```

Comment:
Add date_outcome - both date of death and discharge date are empty in the data (unless Sadie finds a field that isn't empty for this)